tests/cli/invalid/func.wast

```diff
@@ -63,27 +63,27 @@
   )
   "operators remaining after end of function")
 
-(assert_invalid
-  (module
-    (func end unreachable))
+(assert_malformed
+  (module quote
+    "(func end unreachable)")
   "operators remaining after end of function")
 
-(assert_invalid
-  (module
-    (func end return))
+(assert_malformed
+  (module quote
+    "(func end return)")
   "operators remaining after end of function")
 
-(assert_invalid
-  (module
-    (func end br 0))
+(assert_malformed
+  (module quote
+    "(func end br 0)")
   "operators remaining after end of function")
 
-(assert_invalid
-  (module
-    (func end br 100000))
+(assert_malformed
+  (module quote
+    "(func end br 100000)")
   "operators remaining after end of function")
 
-(assert_invalid
-  (module
-    (func end br_table 0))
+(assert_malformed
+  (module quote
+    "(func end br_table 0)")
   "operators remaining after end of function")
```

Comment:
Similar comment on these tests as to the ones above, would you be up for adding a `module binary` dual for these malformed tests?

Reply:
Done.